cli/crates/cli/tests/utils/environment.rs

```diff
@@ -58,52 +56,20 @@ fn get_free_port() -> u16 {
 impl Environment {
     #[allow(clippy::needless_return, clippy::unused_async)]
```

Comment:
Can we get rid of this allow now? (and the others in this file)

Reply:
Definitely, good catch!